airflow/www/templates/airflow/trigger.html

```diff
@@ -44,12 +44,23 @@ <h2>Trigger DAG: {{ dag_id }}</h2>
     <div class="form-group row">
       <div class="col-md-2">
         <label for="run_id">Run id (Optional)</label>
-        <input type="text" class="form-control" placeholder="Run ID" name="run_id" >
+        <input type="text" class="form-control" placeholder="Run ID" name="run_id">
+      </div>
+    </div>
+    <div class="form-group row">
+      <div class="col-md-2">
+        <label for="recent_configs">Recent Configurations</label>
+        <select class="form-control" name="recent_configs" id="recent_configs">
+          <option value="">None</option>
+          {% for conf in recent_confs %}
+          <option value="{{conf}}">{{ conf }}</option>
+          {% endfor %}
+        </select>
       </div>
     </div>
     <label for="conf">Configuration JSON (Optional, must be a dict object)</label>
     <textarea class="form-control" name="conf" id="json">{{ conf }}</textarea>
-    </div>
+
```

Comment:
This div was here to adding some spacing after the json editor. (It wasn't working properly indeed). Maybe adding a line break (<br>), currently it's quite compact :):
![image](https://user-images.githubusercontent.com/14861206/204163363-da614e83-7b2d-4565-aea7-3b699afe0bfc.png)